lib/core/stores/remote.ts

```diff
@@ -496,3 +496,110 @@ function isSearchSchedulesResult(obj: any): obj is { cursor: string; schedules:
     obj.schedules.every(isSchedule)
   );
 }
+
+export class RemoteLockStore implements ILockStore {
+  private url: string;
+  // simple lock counter - tryAcquire increments, release decrements
+  // counter > 0 we call heartbeat api in a loop, setInterval
+  private lockCounter: number = 0;
+  private heartbeatInterval: NodeJS.Timeout | number | null = null;
```

Comment:
```suggestion
  private heartbeatInterval: number | null = null;
```

Let's just coerce this value to a number, you can do this by doing `this. heartbeatInterval  = +setInterval()`